modules/apps/segments/segments-web/test/js/components/criteria_builder/CriteriaRowReadable.es.js

```diff
@@ -0,0 +1,379 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {render} from '@testing-library/react';
+import React from 'react';
+
+import CriteriaRowReadable from '../../../../src/main/resources/META-INF/resources/js/components/criteria_builder/CriteriaRowReadable.es';
+import {dateToInternationalHuman} from '../../../../src/main/resources/META-INF/resources/js/utils/utils.es';
+
+import '@testing-library/jest-dom/extend-expect';
+
+const equalsOperator = {label: 'Equals', name: 'eq'};
+const atLeastOperator = {label: 'At least', name: 'gt'};
+
+const stringCriterion = {
+	operatorName: 'eq',
+	propertyName: 'firstName',
+	value: 'string value',
+};
+
+const stringProperty = {
+	label: 'First Name',
+	name: 'firstName',
+	options: [],
+	type: 'string',
+};
+
+const booleanCriterion = {
+	operatorName: 'eq',
+	propertyName: 'signedIn',
+	value: 'true',
+};
+
+const booleanProperty = {
+	label: 'Signed In',
+	name: 'signedIn',
+	options: [],
+	type: 'boolean',
+};
+
+const dateCriterion = {
+	operatorName: 'eq',
+	propertyName: 'birthDate',
+	value: '2023-09-22',
+};
+
+const dateProperty = {
+	label: 'Date of Birth',
+	name: 'birthDate',
+	options: [],
+	type: 'date',
+};
+
+const entityCriterion = {
+	displayValue: 'Liferay DXP',
+	operatorName: 'eq',
+	propertyName: 'groupIds',
+	unknownEntity: false,
+	value: '20121',
+};
+
+const entityProperty = {
+	label: 'Site',
+	name: 'groupIds',
+	options: [],
+	selectEntity: {
+		id: 'selectEntity',
+		multiple: false,
+		title: 'Select Site',
+		uri: 'http://localhost:8080/test',
+	},
+	type: 'id',
+};
+
+const collectionCriterion = {
+	operatorName: 'eq',
+	propertyName: 'cookies',
+	value: 'key_name=key_value',
+};
+
+const collectionProperty = {
+	label: 'Cookies',
+	name: 'cookies',
+	options: [],
+	type: 'collection',
+};
+
+const doubleCriterion = {
+	operatorName: 'eq',
+	propertyName: 'deviceScreenResolutionHeight',
+	value: '700.00',
+};
+
+const doubleProperty = {
+	label: 'Device Screen Resolution Height',
+	name: 'deviceScreenResolutionHeight',
+	options: [],
+	type: 'double',
+};
+
+const eventCriterion = {
+	operatorName: 'gt',
+	operatorNot: true,
+	propertyName: 'downloadedDocuments',
+	propertyType: 'count',
+
+	query: {
+		conjunctionName: 'and',
+		groupId: 'group_2',
+		items: [
+			{
```

Comment:
With the changes proposed in https://issues.liferay.com/browse/LPS-171785 the right JSON will be something like:

```
    "groupId": "group_0",
    "items": [
      {
        "propertyName": "pageViewed",
        "assetId": "545188693724480037'",
        "day": {
          "value": "2024-01-26T23:00:00.000Z",
          "operator": "eq"
        },
        "operatorName": "ge",
        "value": "1"
      }
    ],
    "conjunctionName": "and"
  }```

Reply:
Yeah, I'm going to adapt the code to those changes